webapp/src/main/webapp/WEB-INF/view/scripts/jquery-ui/js/openmrsSearch.js

```diff
@@ -69,14 +69,18 @@ function OpenmrsSearch(div, showIncludeVoided, searchHandler, selectionHandler,
         opts = {};
     }
 
-    if(!opts.showIncludeVoided)
+    if(!opts.showIncludeVoided) {
```

Comment:
Didn't we need a space between if and bracket?

Reply:
Yeah, I forgot that. Sorry!